sysconfig/jureca-booster/modules.yaml

```diff
@@ -26,17 +26,18 @@ modules:
       suffixes:
           '+profile': 'profile'
           '^coreneuron+knl': 'knl'
+          '~mpi~binary': 'serial'
```

Comment:
minor : why not this just `~mpi` intead of `~mpi~binary`?

Reply:
Your are right. Done!